Header header
uint32 nRightWheelHealthStatus
uint32 nRightWheelDirection
uint32 nRightWheelSpeed
uint32 nLeftWheelHealthStatus
uint32 nLeftWheelDirection
uint32 nLeftWheelSpeed

